src/main/java/com/xiaomi/infra/pegasus/client/PegasusTable.java

```diff
@@ -1761,7 +1761,9 @@ public void handleReplicaException(
     String replicaServer;
     try {
       replicaServer =
-          replicaConfiguration.primary.get_ip() + ":" + replicaConfiguration.primary.get_port();
+          replicaConfiguration.primaryAddress.get_ip()
+              + ":"
+              + replicaConfiguration.primaryAddress.get_port();
```

Comment:
diretly use `toString()`

Reply:
I don't want to change these codes in this pr. Maybe we can change it in later pr.